src/planning/plan/planificator.ts

```diff
@@ -75,9 +68,6 @@ export class Planificator {
     }
     this.consumer = new PlanConsumer(this.arc, this.result);
 
-    this.lastActivatedPlan = null;
-    this.arc.registerInstantiatePlanCallback(this.arcCallback);
```

Comment:
this code seems to no longer be needed in shell, also this is the only use of `un/registerInstantiatePlanCallback` in arc.ts, so it should be ok to remove that too, i think.
thanks for cleaning this up!

Reply:
in web-planner there is a reference to this method 
`this._plansChanged(current, planificator.getLastActivatedPlan())`
the actual plan isn't really used, but worth removing.

Reply:
super, applying suggested fixes.